csep/utils/readers.py

```diff
@@ -702,44 +719,182 @@ def _query_bsi(start_time, end_time, min_magnitude=2.50,
                        minlatitude=min_latitude, maxlatitude=max_latitude,
                        minlongitude=min_longitude, maxlongitude=max_longitude,
                        maxdepth=max_depth,
-                       starttime=start_time, endtime=end_time, **extra_bsi_params)
+                       starttime=start_time, endtime=end_time,
+                       **extra_bsi_params)
 
     return eventlist
-def _parse_datetime_to_zmap(date, time):
-        """ Helping function to return datetime in zmap format.
+
+
+def _query_gcmt(out_format='QuakeML',
```

Comment:
I think this is now conflicting with the just merged pull request for the GNS catalog. This function is exactly at the same place as query_gns, and is causing the conflict. Could you resolve the conflict using the updated version in the master branch? Once that is done, I'll take another look to confirm everything is good and then merge this into master for the upcoming release.

Let me know if you have any questions.

Thanks!!!

Reply:
Thanks!